queries.sql

```diff
@@ -1,3 +1,17 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE '%mon';
+
+SELECT name from animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
```

Comment:
- [ ] As pointed out by the previous reviewer, the query says between 2016 and 2019, which is the beginning of 2016 to the end of 2019. At the moment, you query starts at the beginning of 2016 and stops at the beginning of 2019. Please kindly ensure it runs to the end of 2019.

`BETWEEN 2016-01-01 AND 2019-12-31`